### PR TITLE
Fix glbc downgrade image name in e2e framework

### PR DESCRIPTION
**Pull request by rramkumar1**
This should be working, not sure why not but I am seeing ImagePullBackOff errors in the e2e tests so removing it for now.

**Release note**:
```release-note
None
```

/assign @bowei

### CODE REVIEW

**Review by cblecker — CHANGES_REQUESTED** [review]

File: test/e2e/framework/nodes_util.go [hunk 1]
```diff
@@ -86,7 +86,7 @@ func ingressUpgradeGCE(isUpgrade bool) error {
 		}
 	} else {
 		// Downgrade to latest release image.
-		command = "sudo sed -i -re 's/(image:)(.*)/\\1 k8s.gcr.io\\/google_containers\\/glbc:0.9.7/' /etc/kubernetes/manifests/glbc.manifest"
+		command = "sudo sed -i -re 's/(image:)(.*)/\\1 gcr.io\\/google_containers\\/glbc:0.9.7/' /etc/kubernetes/manifests/glbc.manifest"
```
[the remark this answers is not in the document]
Comment (rramkumar1) [reply test/e2e/framework/nodes_util.go]: Ah I see, done.

**Review by cblecker — APPROVED** [review]

**Review by MrHohn — APPROVED** [review]
/lgtm

File: test/e2e/framework/nodes_util.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (BenTheElder) [reply test/e2e/framework/nodes_util.go]: probably should update the PR title now?

**Comment by cblecker** [comment]
@rramkumar1 can you please fix the commit message?

**Comment by rramkumar1** [comment]
/close
Fixing in another PR

**Pull request closed without merging**